src/bin/common.rs

```diff
@@ -735,6 +742,12 @@ fn parse_config(matches: &ArgMatches) -> Result<EncoderConfig, CliError> {
   if cfg.tune == Tune::Psychovisual {
     cfg.speed_settings.transform.tx_domain_distortion = false;
   }
+  if cfg.tune == Tune::Psnr {
+    cfg.aq_strength = 0.0;
+  }
+  if let Some(aq_str) = matches.value_of("AQ_STRENGTH") {
```

Comment:
Should this be possible in Tune::Psnr is requested?

Reply:
It would be weird. Typically in encoders, the finer grained options override broader options such as presets and tunes, so with that idea I decided that `--aq-strength` should take precedence. But maybe it should be disallowed? And print a warning to the user?

Reply:
Letting AQ_STRENGTH override the broader tune makes sense to me, especially if there's precedent. It could also be documented in the argument definition or program help?